lib/run-tests/ignored-tests.js

```diff
@@ -1,5 +1,4 @@
 module.exports = function shouldIgnore(test) {
-  if (test.file.startsWith("test/intl402")) return true;
-  if (test.attrs.features && test.attrs.features.includes("Proxy")) return true;
+  if (!test.file.startsWith("test/language")) return true;
```

Comment:
Hopefully this will make the test suite a lot faster.

Reply:
I know right! I was thinking the same thing!